clientv3/maintenance.go

```diff
@@ -50,6 +51,11 @@ type Maintenance interface {
 	// Status gets the status of the endpoint.
 	Status(ctx context.Context, endpoint string) (*StatusResponse, error)
 
+	// HashKV returns the state of backend database at the time of the RPC.
+	// If revision is non-zero, hash is computed only in key bucket and keys at or below
```

Comment:
// If revision is zero, the hash is computed on all keys. If the revision is non-zero, the hash is computed on all keys at or below the given revision.